utils.go

```diff
@@ -0,0 +1,7 @@
+package memphis_kafka
+
+import "fmt"
+
+func memphisKafkaErr(msg string) {
```

Comment:
in general besides the case in which the connection with memphis has failed we are not printing anything to the stdout, for now let's create an error subject to which errors will be sent